app/uk/gov/hmrc/eusubsidycompliancefrontend/services/EligibilityJourney.scala

```diff
@@ -72,15 +73,15 @@ object EligibilityJourney {
 
   // TODO - consider introducing form classes for each page
   object FormUrls {
-    val CustomsWaivers = "do-you-claim-customs-waivers"
-    val WillYouClaim = "will-you-claim-customs-waivers"
-    val NotEligible = "not-eligible"
-    val MainBusinessCheck = "main-business-check"
-    val SignOut = "not-eligible-to-lead"
-    val AcceptTerms = "terms-conditions"
-    val EoriCheck = "eoricheck"
-    val SignOutBadEori = "incorrect-eori"
-    val CreateUndertaking = "create-undertaking"
+    val CustomsWaivers = routes.EligibilityController.getCustomsWaivers().url
```

Comment:
Good to see this. I'll pick up the rest of these as part of my work for `ESC-344`.

Reply:
I am getting some weird test case result. Will let you know if that goes well

Reply:
this one is sorted , we can do this in other journeys :)